src/scene/procedural.js

```diff
@@ -639,13 +639,19 @@ pc._createConeData = function (baseRadius, peakRadius, height, heightSegments, c
  */
 pc.createCylinder = function (device, opts) {
     // Check the supplied options and provide defaults for unspecified ones
-    var baseRadius = opts && opts.baseRadius !== undefined ? opts.baseRadius : 0.5;
+    // #ifdef DEBUG
+    if (opts.hasOwnProperty('baseRadius') && !opts.hasOwnProperty('radius'))
+      console.warn('DEPRECATED: "baseRadius" in arguments, use "radius" instead');
+    // #endif
+    opts.radius = opts.radius || opts.baseRadius;
```

Comment:
Reading bellow on other properties, this will throw exception, as `opts` is optional. Check how `height` is defined. So you would need to take in account that scenario.